.github/workflows/lint-prs.yml

```
@@ -0,0 +1,35 @@
+
+      - name: Install commitlint
+        run: |
+          npm install --global @commitlint/cli @commitlint/config-conventional
+
+      - name: Check if PR title changed
+        id: titlecheck
+        run: |
+          echo "TITLE_CHANGED=${{ github.event.changes.title != null }}" >> $GITHUB_ENV
+
+      - name: Lint PR title
+        if: env.TITLE_CHANGED == "true"
```

actionlint (1.7.4)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)